dwave/optimization/model.pyx

```diff
@@ -427,6 +427,7 @@ cdef class Model:
     def into_file(self, file, *,
                   Py_ssize_t max_num_states = 0,
                   bool only_decision = False,
+                  bool compress = False,
```

Comment:
How about we expose compression level as well?

Reply:
I thought about that, but erred on consistency with https://github.com/dwavesystems/dimod/pull/1296 and the others for compatibility. No one has requested finer control for those so :shrug:

Reply:
Makes sense. I can see pros and cons of both approaches.

Reply:
Looks like someone already [did the analysis](https://github.com/dwavesystems/dimod/pull/1296#discussion_r1029893021). :facepalm: